Specs/Scene/PointCloud3DTileContentSpec.js

```diff
@@ -920,21 +901,30 @@ defineSuite([
             var color;
             expect(scene).toRenderAndCall(function(rgba) {
                 color = rgba;
+                console.log(rgba);
             });
 
             tileset.clippingPlanes = new ClippingPlaneCollection ({
                 planes : [
-                    new Plane(Cartesian3.UNIT_Z, -10.0),
+                    new Plane(Cartesian3.UNIT_Z, 0.0),
                     new Plane(Cartesian3.UNIT_X, 0.0)
                 ],
                 modelMatrix : Transforms.eastNorthUpToFixedFrame(tileset.boundingSphere.center),
                 unionClippingRegions: true
             });
 
+            expect(scene).toRenderAndCall(function(rgba) {
+                console.log(rgba);
+            }); // TODO: look at this in Sandcastle
+
             expect(scene).notToRender(color);
 
             tileset.clippingPlanes.unionClippingRegions = false;
 
+            expect(scene).toRenderAndCall(function(rgba) {
+                console.log(rgba);
+            });
```

Comment:
What's with all the `console.logs` in here?

Reply:
this might be a "whoops" sort of thing...